gostream/frontend/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Just a quick question on what to do about the gostream frontend. Should we now just use `local-rc` for such purposes? I do not see a build target for it in the rdk Makefile.

Reply:
Good question. I never actually used Gostream's front end and am not sure why we'd need it given that the viam-server has one. Do you find it useful? If so, I could keep it around, but if not, I'll just get rid of it.

Reply:
I do not use it either - sounds good to remove it.